ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh

```diff
@@ -78,4 +81,4 @@ sshpass -p "$(cat /secret/login)" ssh -oStrictHostKeyChecking=no -oUserKnownHost
    cd jetlag
    git branch
    source bootstrap.sh
-   ansible-playbook -i ansible/inventory/telco_${TYPE}.inv ansible/${TYPE}-deploy.yml -v | tee /tmp/ansible-${TYPE}-deploy-$(date +%s)"
\ No newline at end of file
+   ansible-playbook -i ansible/inventory/cloud31.local ansible/${TYPE}-deploy.yml -v | tee /tmp/ansible-${TYPE}-deploy-$(date +%s)"
```

Comment:
lets make this a var - `CLOUD_ID` so we can introduce other clouds in the future.

Reply:
Done, thanks for the catch